consensus/block_validation_test.go

```diff
@@ -102,6 +102,7 @@ func TestBlockValidSyntax(t *testing.T) {
 		StateRoot: validSt,
 		Miner:     validAd,
 		Ticket:    validTi,
+		Height:    1,
```

Comment:
if unset the default value for height is 0, setting it to 1 as we don't want to treat this as a genesis block.